packages/mongodb-chatbot-server/src/routes/responses/createResponse.ts

```diff
@@ -64,7 +65,20 @@ const CreateResponseRequestBodySchema = z.object({
           z.object({
             type: z.literal("message").optional(),
             role: z.enum(["user", "assistant", "system"]),
-            content: z.string(),
+            content: z.union([
+              z.string(),
+              z
+                .array(
+                  z.object({
+                    type: z.literal("input_text"),
+                    text: z.string(),
+                  })
+                )
+                .refine(
+                  (content) => content.length === 1,
+                  ERR_MSG.INPUT_TEXT_ARRAY
+                ),
```

Comment:
simpler appraoch:

```suggestion
                .length(1, ERR_MSG.INPUT_TEXT_ARRAY),
```

Reply:
oh sweet!

Reply:
oh man, this is relevant to like every single `refine`